src/watertap_contrib/seto/unit_models/surrogate/lt_med_surrogate.py

```diff
@@ -275,6 +275,22 @@ def eq_brine_vol_flow(b):
                 - b.distillate_props[0].flow_vol_phase["Liq"]
             )
 
+        """
+        Isobaric
+        """
+
+        @self.Constraint(self.flowsheet().config.time, doc="Isobaric")
+        def eq_feed_to_distillate_isobaric(b, t):
+            return b.feed_props[t].pressure == b.distillate_props[t].pressure
+
+        @self.Constraint(self.flowsheet().config.time, doc="Isobaric")
+        def eq_feed_to_brine_isobaric(b, t):
+            return b.feed_props[t].pressure == b.brine_props[t].pressure
+
+        @self.Constraint(self.flowsheet().config.time, doc="Isobaric")
+        def eq_feed_to_cooling_isobaric(b, t):
+            return b.feed_props[t].pressure == b.cooling_out_props[t].pressure
```

Comment:
I definitely see how this would be true (assuming equal pressure between feed, distillate and cooling source), but I can also see scenarios where this might not be the case. @Zhuoran29 - please confirm whether this is always the case for PSA's LT-MED model, or if you think there would be any reason to later do sensitivity analysis or just be able to have different pressures between these streams. If there is a reasonable chance that someone might want to do this for whatever reason, let's post an issue about it (i.e., LT-MED assumes isobaric conditions but can later add a config option that allows user to specify different pressures as needed at each of the stateblocks). Either way, I won't let it hold up this PR, but I'd like your feedback on it.

Reply:
Let's try to also consider this from a flowsheet perspective, where the inlet pressure to LT-MED could be different if receiving a stream at a different pressure from atmospheric, for example.

Reply:
@adam-a-a Yes, I confirm it is always the case in this configuration. The surrogate model was developed based on a very specific configuration (actually a bit different than the PSA pilot plant, and is believed better), and in that configuration the pressure are assumed the same across the flows.

So if we're going to develop a more detailed physical model in the future, we should make the pressure more flexible as you mentioned, but at least for the surrogate it shouldn't be changed, otherwise the surrogate equations won't be suitable.